pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/controller/PinotControllerTransport.java

```diff
@@ -51,14 +49,14 @@ public class PinotControllerTransport {
   private final String _scheme;
   private final AsyncHttpClient _httpClient;
 
-  public PinotControllerTransport(Map<String, String> headers, String scheme, @Nullable SSLContext sslContext,
+  public PinotControllerTransport(Map<String, String> headers, String scheme, @Nullable SslContext sslContext,
       ConnectionTimeouts connectionTimeouts, TlsProtocols tlsProtocols, @Nullable String appId) {
```

Comment:
since this is public API we might need to keep supporting both javax.net.ssl.SSLContext and io.netty.handler.ssl.SslContext and mark the former with deprecated annotation.